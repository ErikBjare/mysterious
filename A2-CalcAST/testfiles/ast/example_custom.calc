let
	radius = ask user [2.0]
	badpie = 3.0
in
	2.0 * badpie * radius
end
