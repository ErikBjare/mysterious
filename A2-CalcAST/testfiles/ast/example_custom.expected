Program
  Let
    List
      Binding
        IdDecl ID="radius"
        Ask
          Opt
            Numeral NUMERAL="2.0"
      Binding
        IdDecl ID="badpie"
        Numeral NUMERAL="3.0"
    Mul
      Numeral NUMERAL="2.0"
      Mul
        IdUse ID="badpie"
        IdUse ID="radius"
